### PR TITLE
Fix support of PHP CS Fixer 3.0

### PR DESCRIPTION
**Pull request by StudioMaX**
## Contribution type

Bug fix

## Description of change

This adds support of PHP CS Fixer 3.0.

### CODE REVIEW

File: docs/en/plugins/php_cs_fixer.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ Configuration
 
 * **verbose** [bool, optional] - Whether to run in verbose mode (default: false).
 * **diff** [bool, optional] - Whether to run with the `--diff` flag enabled (default: false).
-* **rules** [string, optional] - Fixer rules (default: `@PSR2`).
+* **rules** [string, optional] - Fixer rules (default: `@PSR12`). Cannot be used simultaneously with the `config` option.
```
Comment (corpsee) [inline docs/en/plugins/php_cs_fixer.md:13]: I guess it should be `@PSR12` for version `3.x` and `@PSR2` for version `2.x`.

**Review by corpsee — CHANGES_REQUESTED** [review]

**Comment by corpsee** [comment]
@StudioMaX Thanks! Could you fix documentation part about default `rules`? Then I can merge PR.

**Comment by corpsee** [comment]
@StudioMaX Could you also rebase your PR on branch `release-1.3` instead of `master` please?

**Comment by StudioMaX** [comment]
@corpsee Fixed this. Unfortunately I can't check this manually on 1.3, since I'm already on 2.0.

**Comment by corpsee** [comment]
Thanks! Version `1.3` I will test it myself. :+1:

**Pull request merged**